docs/content/preview/manage/upgrade-deployment.md

```diff
@@ -121,3 +121,8 @@ Upgrading the YSQL system catalog is an online operation and doesn't require sto
 {{< note title="Note" >}}
 Concurrent operations in a cluster can lead to various transactional conflicts, catalog version mismatches, and read restart errors. This is expected, and should be addressed by re-running `upgrade_ysql`.
 {{< /note >}}
+
+{{< note title="Note" >}}
+Downgrades are currently unsupported. Please see issue [#13686](https://github.com/yugabyte/yugabyte-db/issues/13686) on Github.
```

Comment:
It's better not to have several note elements one right after the other. I suggest just unwrapping both notes (leave the text).
Also, suggestion:
Downgrades are not currently supported.